docs/backend-requests/handling/manual-jwt.mdx

```diff
@@ -9,94 +9,4 @@ Your Clerk-generated [session tokens](/docs/backend-requests/resources/session-t
-
-If the above process is successful, it means that the user is signed in to your application and you can consider them authenticated. You can also retrieve the session ID and user ID out of the token's claims.
-
-</Steps>
-
-## Example usage
-
-In the example below, the [jsonwebtoken](https://www.npmjs.com/package/jsonwebtoken) library is used to verify the token signature. The [cookies](https://www.npmjs.com/package/cookies) library is used to retrieve the `__session` cookie.
-
-```tsx filename="pages/api/token-example.ts"
-import type { NextApiRequest, NextApiResponse } from "next";
-import Cookies from "cookies";
-import jwt from "jsonwebtoken";
-
-export default async function (req: NextApiRequest, res: NextApiResponse) {
-  const publicKey = process.env.CLERK_PEM_PUBLIC_KEY;
-  const cookies = new Cookies(req, res);
-  const sessToken = cookies.get("__session");
-  const token = req.headers.authorization;
-
-  if (sessToken === undefined && token === undefined) {
-    res.status(401).json({ error: "not signed in" });
-    return;
-  }
-
-  try {
-    let decoded = "";
-    if (token) {
-      decoded = jwt.verify(token, publicKey);
-      res.status(200).json({ sessToken: decoded });
-      return;
-    } else {
-      decoded = jwt.verify(sessToken, publicKey);
-      res.status(200).json({ sessToken: decoded });
-      return;
-    }
-  } catch (error) {
-    res.status(400).json({
-      error: "Invalid Token",
-    });
-    return;
-  }
-}
-```
-
-{/* TODO: add code examples of how to validate the exp and azp claims ?? */}
-If the token is valid, the response will return a JSON object that looks similar to the one below. You can use this to validate the `exp` and `azp` claims, as mentioned in the [steps above](#verify-the-token-signature). Once the token is completely validated, you then have a valid session ID (`sid`) and user ID (`sub`) that you can use in your application.
-
-```json
-{
-  "sessToken": {
-    "azp": "http://localhost:3000",
-    "exp": 1687906422,
-    "iat": 1687906362,
-    "iss": "https://magical-marmoset-51.clerk.accounts.dev",
-    "nbf": 1687906352,
-    "sid": "sess_2Ro7e2IxrffdqBboq8KfB6eGbIy",
-    "sub": "user_2RfWKJREkjKbHZy0Wqa5qrHeAnb"
-  }
-}
-```
+The `authenticateRequest()` method from Clerk's JavaScript Backend SDK does all of this for you. It accepts the request object and authenticates the session token in it. See the [reference page](/docs/references/backend/sessions/authenticate-request) for example usage and more information.
```

Comment:
I have some notes that should apply before merging this PR.
- I believe that `authenticate-request` should not be part of the `/docs/references/backend/sessions` as it's not exactly related to  the `sessions` and it's definitely not related to our backend endpoints grouped in that url path.
- code example in [authenticate request page](https://clerk.com/docs/references/backend/sessions/authenticate-request) should be updated to use `@clerk/backend` package instead of nextjs
- I think we should keep a code example of how `authenticateRequest()` can be used in this page, otherwise does this page make much sense? It's just an intro and a ref to another page.

cc: @alexisintech , @octoper

Reply:
yeah I agree - this page will definitely get reworked in the upcoming IA. for now, I've made the changes you requested :)